components/sidebar/SidebarElement.vue

```diff
@@ -2,15 +2,24 @@
 import { version } from "~/package.json";
 
 const { data: collections } = usePrivatePlaylists();
+const isElectron = window.isElectron || false;
```

Comment:
What about using `runtimeConfig.public.systemName` instead of introducing yet another variable..?

Reply:
Didn't know about that one. Sounds like a better idea.